### DISCUSSION TITLE
Documentation Confusing point, "Disable Gateway"

### DISCUSSION THREAD

**Discussion by adamsd5** [zwave-js/zwave-js-ui#3419] [General]
The complete documentation for "Disable Gateway" says this:

> [Disable Gateway](https://zwave-js.github.io/zwave-js-ui/#/usage/setup?id=disable-gateway)
> Enable this to use Z-Wave JS UI as only a Control Panel

I have no idea what this means.  If the switch is "On", does that mean it is "Disabled"?  So if I turn the switch to "off", is that "Enabled" so that I can "use the Z-Wave JS UI as only a control panel"?

Further, what exactly does it mean to "use Z-Wave JS UI s only a Control Panel"?  

I got here because I'm trying to setup the ZWave JS UI for the first time.  I have my Zooz USB stick and two Zooz thermometers showing up in the UI, but not as devices that I can use elsewhere in Home Assistant.  I think this option is relevant, but I don't know if I'm supposed to have it turned on or off.

**Comment by kpine** [comment]
> I have no idea what this means. If the switch is "On", does that mean it is "Disabled"? So if I turn the switch to "off", is that "Enabled" so that I can "use the Z-Wave JS UI as only a control panel"?

Gateway refers to the MQTT gateway. If you don't need MQTT functionality, you can disable the MQTT gateway.

> Further, what exactly does it mean to "use Z-Wave JS UI s only a Control Panel"?

It means you aren't using MQTT, you use the Control Panel UI only.

> I got here because I'm trying to setup the ZWave JS UI for the first time. I have my Zooz USB stick and two Zooz thermometers showing up in the UI, but not as devices that I can use elsewhere in Home Assistant. I think this option is relevant, but I don't know if I'm supposed to have it turned on or off.

There is a dedicated installation section for the HA Z-Wave integration: https://zwave-js.github.io/zwave-js-ui/#/homeassistant/homeassistant-official?id=z-wave-integration. You can leave MQTT on if you want to augment the integration support, but most people turn it off.
